populate/src/test/java/bio/terra/pearl/populate/PopulateDemoTest.java

```diff
@@ -66,12 +75,71 @@ private void checkOldVersionEnrollee(List<Enrollee> sandboxEnrollees) {
         );
     }
 
+    /** confirm the proxy enrollee with one governed user was enrolled appropriately */
+    private void checkProxyWithOneGovernedEnrollee(List<Enrollee> sandboxEnrollees) {
+        Enrollee governedEnrollee = sandboxEnrollees.stream().filter(sandboxEnrollee -> "HDGOVR".equals(sandboxEnrollee.getShortcode()))
+                .findFirst().get();
+
+        Enrollee proxyEnrollee = sandboxEnrollees.stream().filter(sandboxEnrollee -> "HDPROX".equals(sandboxEnrollee.getShortcode()))
+                .findFirst().get();
+        assertThat(enrolleeRelationService.findByTargetEnrolleeId(governedEnrollee.getId()), hasSize(1));
+        assertThat(enrolleeRelationService.findByTargetEnrolleeId(governedEnrollee.getId()).get(0).getEnrolleeId(), equalTo(proxyEnrollee.getId()));
+        List<PortalParticipantUser> portalParticipantUsers = portalParticipantUserService.findByParticipantUserId(proxyEnrollee.getParticipantUserId());
+        assertThat(portalParticipantUsers, hasSize(1));
+        assertThat(portalParticipantUsers.get(0).getParticipantUserId(), equalTo(proxyEnrollee.getParticipantUserId()));
+        List<PortalParticipantUser> governedPortalParticipantUsers = portalParticipantUserService.findByParticipantUserId(governedEnrollee.getParticipantUserId());
+        assertThat(governedPortalParticipantUsers, hasSize(1));
+        assertThat(governedPortalParticipantUsers.get(0).getParticipantUserId(), equalTo(governedEnrollee.getParticipantUserId()));
+        PortalParticipantUser proxyPPUser = portalParticipantUsers.get(0);
+        PortalParticipantUser governedPPUser = governedPortalParticipantUsers.get(0);
+        Profile proxyProfile = profileService.find(proxyPPUser.getProfileId()).get();
+        Profile governedUserProfile = profileService.find(governedPPUser.getProfileId()).get();
+        assertThat(proxyProfile.getContactEmail(), equalTo(governedUserProfile.getContactEmail()));
+        assertThat(proxyProfile, not(equalTo(governedUserProfile)));
+    }
+
+    /** confirm the proxy enrollee with two governed users was enrolled appropriately */
+    private void checkProxyWithTwoGovernedEnrollee(List<Enrollee> sandboxEnrollees) {
```

Comment:
same as above -- this is a great test, just break it into sections with comments to make it more readable